packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,22 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map ${searchQueue.length ? 'or in these directories:' : ''}\n` +
```

Comment:
*technically*, it hasn’t to be a directory (can be a file of that name), but I guess nobody has files in `node_modules` these days.